api/controllers/apps_test.go

```diff
@@ -47,7 +49,10 @@ func TestAppShow(t *testing.T) {
 }
 
 func TestAppShowWithAppNotFound(t *testing.T) {
-	aws := test.StubAws(test.DescribeStackNotFound("bar"))
+	aws := test.StubAws(
+		test.DescribeStackNotFound("bar"),
```

Comment:
I think you are right. I actually had it this way to begin with, and changed it for some raisin... I think it had something to do with preventing duplicates, but whatever the reason it's a non-issue now. I'll try reversing the order.